packages/blockchain-wallet-v4-frontend/src/scenes/Login/components/UrlNoticeBar/index.tsx

```diff
@@ -5,14 +5,11 @@ import styled from 'styled-components'
 import { Icon, Link, Text } from 'blockchain-info-components'
 import { media } from 'services/styles'
 
-const BACKGROUND_COLOR = '#0C6CF2'
-const FONT_COLOR = 'white'
 const BC_URL = 'https://login.blockchain.com'
```

Comment:
this should come from the config

Reply:
Sorry pedro but I couldn't find any example, if you do please let me know. For now I'll move forward according to andrews suggestion. Thanks again.

<img width="633" alt="image" src="https://user-images.githubusercontent.com/97299628/154313060-39b06a8d-ebaf-4fc1-956f-99c01a39e2cd.png">